docs/user/index.md

```diff
@@ -0,0 +1,667 @@
+# User & Programmers Manual
```

Comment:
This file self half-done and with marks in Spanish. It should be reviewed.

Reply:
Ok. Fixed in 929b2d68c7924388ce8cab299dc72bf55e459cb5